docs/installation/prepare/microcontrollers/_index.md

```diff
@@ -26,361 +26,245 @@ Recommended configuration: 384kB Ram + 8MB SPIRAM + 4MB Flash
-    --disable-capstone --disable-vnc \
-    --disable-sdl --disable-gtk --extra-cflags="-I/opt/homebrew/Cellar/libgcrypt/1.10.1/include -I/opt/homebrew//include/"
-cd build && ninja
+canon bash -lc "make build-esp32-bin"
+make flash-esp32-bin
 ```
 
 {{% /tab %}}
-{{% tab name="Linux" %}}
-
-On Ubuntu or Debian, first make sure you have the `libgcrypt` library and headers installed by running the following command:
-
-```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-sudo apt-get install libgcrypt20 libgcrypt20-dev
-```
+{{% tab name="Local environment" %}}
 
-Then, run the following command to install QEMU:
+Make sure you have sourced `. ~/dev/esp/export-rs.sh` and `. ~/dev/esp/esp-idf/export.sh` prior running the following command
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-git clone https://github.com/espressif/qemu
-cd qemu
-./configure --target-list=xtensa-softmmu     --enable-gcrypt \
-    --enable-debug --enable-sanitizers  --disable-strip --disable-user \
-    --disable-capstone --disable-vnc --disable-sdl --disable-gtk
-cd build && ninja
+make upload
 ```
 
 {{% /tab %}}
 {{% /tabs %}}
 
-Add `export QEMU_ESP32_XTENSA=<path-to-clone-qemu>/build/` to your `.zshrc` or `.bashrc`, or save this command to run in your terminal every session you wish to use the QEMU emulator.
-
-### Build with the QEMU ESP32 Emulator
-
-Navigate to the root of the Micro-RDK repository.
-Once you've `cd`'d to the correct repository, run `. $HOME/esp/esp-idf/export.sh` if you haven't done so already in this terminal session.
-
-You will need to comment out two lines from the file `sdkconfig.defaults`:
+While running above commands, you may be presented with an interactive menu of different serial port options to use to connect to the ESP32 board.
 
-``` editorconfig
-CONFIG_ESPTOOLPY_FLASHFREQ_80M=y
-CONFIG_ESPTOOLPY_FLASHMODE_QIO=y
-```
-
-You can then run:
-
-```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-make sim-local
-```
-
-Or, if you want to connect a debugger:
+If successful you will retain a serial connection to the board until `Ctrl-C` is pressed.
+To manage this connection, consider running it within a dedicated terminal session, or under `tmux` or `screen`.
+While the serial connection is live, you can also restart the currently flashed image with `Ctrl-R`.
 
-```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-make debug-local
-```
+If everything went well, your ESP32 will be programmed so that you will be able to see your robot live and connect to it on [the Viam app](https://app.viam.com).
 
 ### Troubleshooting
 
-If you are unable to connect to the ESP32-backed robot as a remote, try adding `:4545` to the end of the value set in the remote's `Address` field.
+If you run into the following error `Failed to open serial port` when flashing your esp32 with linux, make sure the user is added to the group `dialout` with `sudo gpasswd -a $USER dialout`

```

Comment:
```suggestion
If you run into the error `Failed to open serial port` when flashing your ESP32 with Linux, make sure the user is added to the group `dialout` with `sudo gpasswd -a $USER dialout`
```

Also what user? This seems like more info could help but I haven't done this step so it might be super obvious 😆 (opt)

Reply:
this is a shortcut to the current user in bash felt a bit easier that to suggest to substitute the username in the command